frontend/apps/docs/content/docs/web.mdx

```diff
@@ -2,25 +2,48 @@
 title: Liam ERD Web
 ---
 
-TBD
+Liam ERD Web is an online tool hosted by `@liam-hq` that allows you to explore real-world ER diagrams without needing to install a CLI! 🛸💫
 
-## Detect schema format
+It enables you to view ER diagrams generated from publicly available schema files, such as `db/schema.rb`, `prisma.schema`, or SQL files derived from `pg_dump`.
 
-You can instantly generate an ER diagram by appending a GitHub file URL to `liambx.com/erd/p/`. For example:
+## How to Explore 🛸💫
 
-https://liambx.com/erd/p/github.com/mastodon/mastodon/blob/main/db/schema.rb
+Inserting `liambx.com/erd/p/` into your browser's address bar will generate an ER diagram for your schema file.
 
-When Liam ERD fetches your schema file, it automatically attempts to determine the schema format. Here’s how the detection works:
+![Inserting Animation](/images/content/docs/1.gif)
 
-- **File name check**: Certain filenames like `schema.rb` or `Schemafile` are assumed to be **`schemarb`**.
-- **File extension check**: Files ending with **`.rb`** are treated as **`schemarb`**, while files ending with **`.sql`** are treated as **`postgresql`**.
-- For more details, refer to the [detectFormat.ts](https://github.com/liam-hq/liam/blob/main/frontend/packages/db-structure/src/parser/supportedFormat/detectFormat.ts) file in our GitHub repository.
+For example, if the schema file you want to explore is hosted at the following URL:
 
-### Override the Detected Format
-If the automatic detection doesn’t match your desired format, you can specify it manually via query parameters. For example:
+```
+# A public repo's schema file
+https://github.com/docusealco/docuseal/blob/master/db/schema.rb
+```
+
+You can generate an ER diagram by inserting `liambx.com/erd/p/` into the URL:
+
+```
+https://liambx.com/erd/p/github.com/docusealco/docuseal/blob/master/db/schema.rb
+      👾^^^^^^^^^^^^^^^^^👾
```

Comment:
For this page, I thought it would be more fun with lots of emojis, so I added more than on other pages.  

Not just in this line, but there might be a bit too many emojis overall.

Reply:
That's cool 😆
I have difficulty with the tone, but it looks like it would be good to make this page fun! I was thinking that we could at least remove the pictograms in the headline and leave the body text 💭